web/docs/auth/migrate-from-11.md

```diff
@@ -86,51 +142,195 @@ You can read more about the new auth system in the [Auth Entities](./entities) s
 
 ## How to Migrate?
 
+### Migrating your project to the new structure
```

Comment:
Somewhere here at the start of migration instructions, I would provide link to our Discord and tell them if they get stuck on anything they should reach out to us.

Reply:
Sure, I'll add it in